modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+// * microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-dry-run-v2_{context}"]
+= Performing a dry run for V2
+
+You can use oc-mirror to perform a dry run, without actually mirroring any images. This allows you to review the list of images that would be mirrored. A dry run also allows you to catch any errors with your image set configuration early. 
+
+.Procedure
+
+. Run the `oc mirror` command with the `--dry-run` flag to perform a dry run:
++
+[source,terminal]
+----
+$ oc mirror -c <image_set_config.yaml> --from file://<oc-mirror workspace path> docker://<mirror-registry-url> --dry-run --v2 <1> <2>
+----
+<1> Replace <image-set-config.yaml> file with the image set configuration file that was created.
+<2> Specify the oc-mirror workspace path.
+
+.Verification
+
+. Navigate into the workspace directory that was generated:
++
+[source,terminal]
+----
+$ cd <oc-mirror workspace path>
+----
+
+. Review the `mapping.txt` file that was generated. This file contains a list of all images that would be mirrored.
```

Comment:
@subhtk this is incorrect, please help refer here to question no:6 here https://docs.google.com/document/d/1NoRR_pG4W10Q1KiOZ_LaCUuYuAxgVlrv_4BPu-6K-5s/edit#heading=h.vmuf853hmqmh, we discussed this in the meeting last time.

Reply:
Done.